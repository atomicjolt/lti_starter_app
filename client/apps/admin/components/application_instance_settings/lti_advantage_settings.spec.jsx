import React from 'react';
import TestRenderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';

import LtiAdvantageSettings from './lti_advantage_settings';

const mockStore = configureStore([]);
const store = mockStore({});

describe('LtiAdvantageSettings', () => {
  it('renders the LtiAdvantageSettings component', () => {
    const params = {
      applicationId: '1',
      applicationInstanceId: '2',
    };

    const result = TestRenderer.create(
      <Provider store={store}>
        <LtiAdvantageSettings params={params} />
      </Provider>
    );
    expect(result).toMatchSnapshot();
  });
});
