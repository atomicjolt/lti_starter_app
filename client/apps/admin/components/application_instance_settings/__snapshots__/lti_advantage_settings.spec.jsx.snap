// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AtomicLtiSettings renders the AtomicLtiSettings component 1`] = `
<div
  className="aj-columns"
>
  <div
    className="aj-col-flex"
  >
    <div
      className="aj-settings-title"
    >
      <div
        className="aj-col-flex"
      >
         
        LTI Deployments
      </div>
      <div
        className="flex-center"
      >
        <button
          className="aj-btn"
          onClick={[Function]}
        >
          New LTI Deployment
        </button>
      </div>
    </div>
  </div>
</div>
`;
